ocaml/xapi/xapi_vlan.ml

```diff
@@ -27,6 +27,23 @@ let pool_introduce ~__context ~tagged_PIF ~untagged_PIF ~tag ~other_config =
   in
   (* Untagged PIF going to be VLAN_master_of above VLAN *)
   Db.PIF.set_VLAN_master_of ~__context ~self:untagged_PIF ~value:vlan ;
+  (* Ensure that the untagged PIF shares PIF_metrics of tagged PIF.
+   * This is useful for the VLANs created in pool.join for management network.
+   *)
+  let metrics = Db.PIF.get_metrics ~__context ~self:untagged_PIF in
```

Comment:
Would suggest to use names `untagged_pif` and `tagged_pif` or similar - tagged vs untagged is the theme here.